x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/vectors/50_dense_vector_field_usage.yml

```diff
@@ -0,0 +1,87 @@
+setup:
+  - skip:
+      features: headers
+
+  - do:
+      indices.create:
+        index: futest
+        body:
+          settings:
+            routing.rebalance.enable: none
+            index.number_of_shards: 1
+            index.number_of_replicas: 0
+          mappings:
+            properties:
+              name:
+                type: keyword
+              vector:
+                type: dense_vector
+                dims: 5
+                index: true
+                similarity: l2_norm
+
+  - do:
+      index:
+        index: futest
+        body:
+          name: cow.jpg
+          vector: [ 230.0, 300.33, -34.8988, 15.555, -200.0 ]
+          comments:
```

Comment:
The `comments` part of these documents can be removed. I accidentally forgot to clean this up when I simplified the mapping in https://github.com/elastic/elasticsearch/pull/80516.

Reply:
6e42856 👍